### PR TITLE
[show] vnet endpoint [ip/ipv6] command

### PR DESCRIPTION
**Pull request by siqbal1986**
#### What I did
Created a new CLI Command Show vnet endpoint with option to give IPv4 or IPv6 address parameter.

#### How I did it
This command goes over all vxlan tunnels and checks each endpoints monitor session to populate the endpoint state. while doing so it also tracks the prefixes pointing to an endpoint.

#### How to verify it
run **show vnet endpoint** or **show vnet endpoint<ipv4 address/ipv6 address>**.

```
>>show vnet endpoint  
Endpoint               Endpoint Monitor         prefix count  status
---------------------  ---------------------  --------------  --------
fddd:a100:a251::a10:1  fddd:a100:a251::a10:1               1  Unknown
fddd:a101:a251::a10:1  fddd:a101:a251::a10:1               1  Down
100.251.7.1            100.251.7.1                         3  Up


>>show vnet endpoint fddd:a101:a251::a10:1
Endpoint               Endpoint Monitor           prefix                        status
---------------------  -------------------------  ----------------------------  --------
fddd:a101:a251::a10:1  ['fddd:a101:a251::a10:1']  ['fddd:a156:a251::a6:1/128']  Down


>>show vnet endpoint 100.251.7.1
Endpoint     Endpoint Monitor    prefix                                                        status
-----------  ------------------  ------------------------------------------------------------  --------
100.251.7.1  ['100.251.7.1']     ['160.162.191.1/32', '160.163.191.1/32', '160.164.191.1/32']  Up
```
#### Previous command output (if the output of a command-line utility has changed)
None
#### New command output (if the output of a command-line utility has changed)
NA

### CODE REVIEW

File: doc/Command-Reference.md [hunk 1]
```diff
@@ -9517,6 +9517,42 @@ This command displays brief information about all the vnets configured in the de
...
   ```
 
+**show vnet endpoint <ip/ipv6>**
+
+This command displays the list or vxlan tunnel endpoints and their status. In addition it also shows the number of prefixes associated with each endpoints in the tunnels. If an IP address of an endpoint is provided, this command also shows the associated prefixes a well.
+
+- Usage:
+
+  ```
+  show vnet endpoint <ipv4_address/ipv6_address>
```
Comment (prsunny) [inline doc/Command-Reference.md:9527]: Can we also print the "`endpoint_monitor`" ip so we have a mapping? BFD session just have` endpoint_monitor'`

File: show/vnet.py [hunk 2]
```diff
@@ -195,6 +195,84 @@ def neighbors():
...
+            val = appl_db.get_all(appl_db.APPL_DB, k)
+            endpoints = val.get('endpoint').split(',') if 'endpoint' in val else []
+            if 'endpoint_monitor' in val:
+                monitors = val.get('endpoint_monitor').split(',')
+            else:
+                continue
+            for idx, endpoint in enumerate(endpoints):
+                monitor_dict[endpoint] = monitors[idx]
+                if endpoint not in prefix_count:
+                    prefix_count[endpoint] =0
```
Comment (prsunny) [inline show/vnet.py:235]: space after `= `for consistency

File: show/vnet.py [hunk 3]
```diff
@@ -195,6 +195,84 @@ def neighbors():
...
+    else:
+        table = []
+        header = ['Endpoint', 'prefix', 'status']
+        state = 'Unknown'
+        prefix =[]
+        have_status = False
+        for k in vnet_rt_keys:
+            val = appl_db.get_all(appl_db.APPL_DB, k)
+            endpoints = val.get('endpoint').split(',')
+            monitors =  val.get('endpoint_monitor').split(',')
```
Comment (prsunny) [inline show/vnet.py:257]: Extra space after `=`. Please fix spacing in all places

File: doc/Command-Reference.md (hunk 1 above)
Comment (siqbal1986) [reply doc/Command-Reference.md:9527]: ok

File: show/vnet.py (hunk 3 above)
Comment (siqbal1986) [reply show/vnet.py]: done

File: doc/Command-Reference.md (hunk 1 above)
Comment (siqbal1986) [reply doc/Command-Reference.md:9527]: done

**Review by prsunny — APPROVED** [review]

**Pull request merged**